demo/src/components/MainContent.tsx

```diff
@@ -128,7 +128,7 @@ export function MainContent({
                 UI components for embeddable staking
               </p>
             </div>
-            <div className="my-8 mx-auto w-[350px] h-[393px] overflow-hidden border border-[#ddd] rounded-md">
+            <div className="my-8 mx-auto w-[350px] h-[450px] overflow-hidden">
```

Comment:
@guillaumegaluz this is a problematic change - the default wallet connection on ETH needs at least a height of 450px

Reply:
To add, this is obviously not ideal - we just haven't yet figured out "auto-resizing" of the iframe.